packages/default/scss/tabstrip/_layout.scss

```diff
@@ -1,17 +1,28 @@
-$tabstrip-tab-padding-x: $nav-item-padding-x !default;
-$tabstrip-tab-padding-y: $nav-item-padding-y !default;
-$tabstrip-tab-gap: $nav-item-margin !default;
-
 @include exports("tabstrip/layout") {
 
+    .k-tabstrip-wrapper {
+        padding: $tabstrip-wrapper-padding-y $tabstrip-wrapper-padding-x;
+        border-width: $tabstrip-wrapper-border-width;
+        border-style: solid;
+        display: flex;
+        flex-direction: column;
+    }
+
     .k-tabstrip {
         border-width: 0;
+        border-color: transparent;
+        color: inherit;
         background-color: transparent;
```

Comment:
Shouldn't `border-color`, `color` and `background-color` props be in `_theme.scss`?

Reply:
`k-tabstrip` is also `k-widget`, which adds all three: border size and color, background color and text color. We do not want a border around the tabstrip (or if we need to, we can use `k-tabstrip-wrapper`). That's why I am removing it in the layout file.

Note: we can always get away with removing tabstrip wrapper, we just need to fix a thing here and there.